just-install.json

```diff
@@ -619,6 +634,13 @@
         "x86": "http://downloads.atlassian.com/software/sourcetree/windows/SourceTreeSetup_{{.version}}.exe"
       }
     },
+    "speccy": {
+      "version": "1.2.8",
+        "installer": {
+        "kind": "custom",
```

Comment:
Cannot use the type "custom" as is, see other entries for an example of how to use "custom" installer. Additionally, are you sure that this package's installation can't be covered by one of the recognised installer types?